src/Tempest/Log/tests/LogLevelTest.php

```diff
@@ -24,13 +24,13 @@ public static function levelsProvider(): array
     {
         return [
             [Level::Emergency, LogLevel::EMERGENCY],
-            [Level::Alert, LogLevel::ALERT],
-            [Level::Critical, LogLevel::CRITICAL],
-            [Level::Error, LogLevel::ERROR],
-            [Level::Warning, LogLevel::WARNING],
-            [Level::Notice, LogLevel::NOTICE],
-            [Level::Info, LogLevel::INFO],
-            [Level::Debug, LogLevel::DEBUG],
+            [Level::Alert,     LogLevel::ALERT],
+            [Level::Critical,  LogLevel::CRITICAL],
+            [Level::Error,     LogLevel::ERROR],
+            [Level::Warning,   LogLevel::WARNING],
+            [Level::Notice,    LogLevel::NOTICE],
+            [Level::Info,      LogLevel::INFO],
+            [Level::Debug,     LogLevel::DEBUG],
         ];
```

Comment:
This feature of aligning arrays as tables is enabled by default, not sure if we should disable it here or not ( controlled with `array_table_style_alignment` ).

Reply:
I don't mind that. Let's keep it and see if we get issues over time